server/users.ts

```diff
@@ -382,6 +382,7 @@ export class User extends Chat.MessageContext {
 	statusType: StatusType;
 	userMessage: string;
 	lastWarnedAt: number;
+	currentChallengeCount: number;
```

Comment:
I don't think this needs more state; the challenge count should be able to be calculated from existing state (`Ladders.challenges`?)